### DISCUSSION TITLE
How do I use enable3d in a vue3 project

### DISCUSSION THREAD

**Discussion by volcanoqq** [enable3d/enable3d#363] [Help / Questions]
I've used threejs to initialize the scene, how to quickly initialize the physical world and add physical rigid bodies.
When I instantiated AmmoPhysics using @enable3d/ammo-physics I was prompted that Ammo was undefined.
![image](https://github.com/enable3d/enable3d/assets/113505790/56c5d202-17ed-4b49-b8ff-916ad9e94868)
![image](https://github.com/enable3d/enable3d/assets/113505790/62297710-88dc-4bd1-b2ab-ca2c0446e69b)

**Comment by volcanoqq** [comment]
![image](https://github.com/enable3d/enable3d/assets/113505790/44bdbfb6-bd9c-47fb-848d-69ed08dbcc24)
It worked

**Reply by 3261564019** [reply]
My project should be helpful to you

https://gitee.com/explorer_clc/three-js-for-vue.git
